src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/placement.pipe.ts

```diff
@@ -0,0 +1,41 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import _ from 'lodash';
+
+@Pipe({
+  name: 'placement'
+})
+export class PlacementPipe implements PipeTransform {
+  /**
+   * Convert the placement configuration into human readable form.
+   * The output is equal to the column 'PLACEMENT' in 'ceph orch ls'.
+   * @param serviceSpec The service specification to process.
+   * @return The placement configuration as human readable string.
+   */
+  transform(serviceSpec: object | undefined): string {
+    if (_.isUndefined(serviceSpec)) {
+      return $localize`no spec`;
+    }
+    if (_.get(serviceSpec, 'unmanaged', false)) {
+      return $localize`unmanaged`;
+    }
+    const kv: Array<any> = [];
+    const hosts: Array<string> = _.get(serviceSpec, 'placement.hosts');
+    const count: number = _.get(serviceSpec, 'placement.count');
+    const label: string = _.get(serviceSpec, 'placement.label');
+    const hostPattern: Array<string> = _.get(serviceSpec, 'placement.host_pattern');
```

Comment:
The `host_pattern` is a string.

e.g.

Row:
![Screen Shot 2020-11-23 at 4 43 53 PM](https://user-images.githubusercontent.com/1691518/99942771-29ddee80-2dab-11eb-9225-93370b5e20de.png)

Spec:
![Screen Shot 2020-11-23 at 4 43 42 PM](https://user-images.githubusercontent.com/1691518/99942778-2cd8df00-2dab-11eb-8d5f-3024df7722cc.png)

Reply:
Hmmm, need to investigate that what arrives in the UI. In the [orchestrator](https://github.com/ceph/ceph/blob/2f0fe914be27463d7a5a6b392fcb3adc74a50d72/src/python-common/ceph/deployment/service_spec.py#L222) the host is an array.